lua/precognition/init.lua

```diff
@@ -352,9 +352,16 @@ end
 ---@param opts Precognition.PartialConfig
 function M.setup(opts)
     config = vim.tbl_deep_extend("force", default, opts or {})
+    if opts.highlightColor then
+        config.highlightColor = opts.highlightColor
+    end
```

Comment:
```suggestion
```

Is this to ensure that the highlight tables aren't merged? If not, this might be redundant.

Reply:
Nvm this makes sense

Reply:
just for future me or anyone, this is for ensuring that if the user has passed a custom table, we don't inject our default `Comment` link